WIP - Exploration

Carmer opened the pull request:
@bethsebian  - 


1.) Do you have in app in mind for them to clone to look into a rails app? Do we plan for them to spin one up? If so, should we add directions to this file to support the questions?

2.) For question 1 - do you think it would be a good idea to give some pointers of things to look for? Or say "Find at least 10" or something like that. I've seen some students really do the bare minimum and I'd like to avoid an answer like "Rails is bigger".

bethsebian commented:
@Carmer Great suggestions!
1) I mentioned this in my DM -- we'll need to find two apps to link to here. 

2) Yeah, I love the idea of more specificity. I'd recommend something like: "Identify 5 differences, and for each one describe 1-2 implications. What effect does that difference have for each framework"

Ex: Sinatra convention allows you to define all routes in one file. This is probably fine for smaller apps, but if you're dealing with a lot of paths (full CRUD for several models), this file would become long and hard to navigate.

Carmer commented:
@bethsebian 

great suggestions. Checkout what I added and let me know what you think. 

Also, I added one more question directing them to the routes.rb file to be sure they find that difference.

bethsebian commented:
Looks great! 👍
